GUI/sanity-test-halo/cypress/cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
```

Comment:
Codacy has a fix for the issue: [Replace `'./commands'` with `"./commands";`](https://app.codacy.com/gh/Seagate/cortx-experiments/pullRequest?prid=9333958)
```suggestion
import "./commands";
```